hawkular-alerts-actions-plugins/hawkular-alerts-actions-email/src/main/java/org/hawkular/alerts/actions/email/EmailTemplate.java

```diff
@@ -268,25 +264,27 @@ private void subjects(Map<String, Object> props, Alert alert) {
     }
 
     private String oneCondAvailabilityDescription(AvailabilityCondition cond) {
-        if (cond.getOperator().equals(AvailabilityCondition.Operator.UP)) {
-            return "up";
-        } else if (cond.getOperator().equals(AvailabilityCondition.Operator.NOT_UP)) {
-            return "not up";
-        } else if (cond.getOperator().equals(AvailabilityCondition.Operator.DOWN)) {
-            return "down";
+        switch (cond.getOperator()) {
+            case UP:
+                return "up";
+            case NOT_UP:
+                return "not up";
+            case DOWN:
+                return "down";
```

Comment:
Probably, but the context here is to build a text that is show in the email.
I think that for future the best place is to play with i18n as this text for sure will be localized/translated at some point.